integration/tests/cook/test_basic.py

```diff
@@ -1857,6 +1857,50 @@ def test_basic_docker_job(self):
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
         self.assertIn('success', [i['status'] for i in job['instances']])
 
+    @unittest.skipUnless(util.default_submit_pool() is not None, "Test requires a default test pool")
+    def test_request_gpu_models(self):
+        settings_dict = util.settings(self.cook_url)
+        valid_gpu_models_config_map = settings_dict.get("pools", {}).get("valid-gpu-models", [])
+        if valid_gpu_models_config_map:
+            default_pool = os.getenv('COOK_TEST_GPU_POOL') or util.default_submit_pool()
+            matching_gpu_models = [ii["valid-models"] for ii in valid_gpu_models_config_map if
+                            re.match(ii["pool-regex"], default_pool)]
+            if len(matching_gpu_models) == 0:
+                job_uuid, resp = util.submit_job(
+                    self.cook_url, 
+                    gpus=1,
+                    env={'COOK_GPU_MODEL':'nvidia-tesla-p100'})
```

Comment:
Missing space between key and value. You can run this through the python formatter to resolve these issues.